### PR TITLE
CI to use Azdev

### PR DESCRIPTION
**Pull request by tjprescott**
Modifies CI scripts to utilize azdev commands (v0.1.0) in lieu of local scripts.

### CODE REVIEW

File: scripts/ci/precheck_pep8.sh [hunk 1]
```diff
@@ -2,12 +2,7 @@
...
 
-# install flake 8
-pip install -qqq flake8
-
-echo "Run flake8."
-flake8 --statistics --config=./.flake8 
-
-# build packages
-. $(cd $(dirname $0); pwd)/artifacts.sh
+export AZDEV_CLI_REPO_PATH=$(pwd)
```
Comment (marstr) [inline scripts/ci/precheck_pep8.sh]: This line assumes that the script will always be run from the root of the repository. This is likely going to always be true, but I'd still prefer to be defensive and use pushd/popd to truly assert this.

For instance:


``` bash
#!/usr/bin/env bash
set -ev
pushd "$(dirname ${BASH_SOURCE[0]})/../.." > /dev/null 
azdev style --pep7
popd > /dev/null
```

Comment (marstr) [reply scripts/ci/precheck_pep8.sh]: Note, this comment is applicable a few places.

Comment (marstr) [reply scripts/ci/precheck_pep8.sh]: You know, given that there's so little in these scripts, you may as well skip this. When we move this over to ADO, we may consider just invoking azdev directly and deleting these scripts all-together.

**Comment by tjprescott** [comment]
I'm going to close this and reopen with an upstream branch.

**Pull request closed without merging**